openrouteservice-api-tests/src/test/java/heigit/ors/services/matrix/ResultsValidationTest.java

```diff
@@ -21,6 +21,7 @@
 package heigit.ors.services.matrix;
 
 import static io.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.*;
```

Comment:
Do any objects from this package get used? If not then it shouldn't be imported.